sonar-java-plugin/src/main/resources/static/documentation.md

```diff
@@ -183,7 +183,7 @@ class A {
         @Deprecated
         List<Tree> getMethodComplexityNodes(ClassTree enclosingClass, MethodTree methodTree);
         ```
-    * `org.sonar.plugins.java.api.JavaResourceLocator`: The following method has been dropped (deprecated since SonarJava 4.1), without replacement.
+    * `org.sonar.plugins.java.api.JavaResourceLocator`: The following method has been dropped (deprecated since VersiOn 4.1), without replacement.
```

Comment:
VersiOn ? 

```suggestion
    * `org.sonar.plugins.java.api.JavaResourceLocator`: The following method has been dropped (deprecated since version 4.1), without replacement.
```